testpilot/metrics/views.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.conf import settings
+from django.http import Http404
+
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.response import Response
+from rest_framework.decorators import permission_classes, api_view
+
+DEFAULT_LOG_PING_WHITELIST = [
+    'testpilottest'
+]
+
+
+@csrf_exempt
+@api_view(['POST'])
+@permission_classes([])
+def log_ping(request, logger_name):
+    """Accept and log metrics pings"""
+    whitelist = getattr(settings, 'LOG_PING_WHITELIST', DEFAULT_LOG_PING_WHITELIST)
```

Comment:
FWIW, the whitelist is kind of a leftover from when we thought we'd be doing more of our own metrics with events like `newuser` and `newinstall`